packages/monitor-v2/src/monitor-polymarket/MonitorProposalsOrderBook.ts

```diff
@@ -79,6 +79,10 @@ async function processProposal(proposal: OptimisticPriceRequest, params: Monitor
       loserOutcome = 1 - winnerOutcome;
     } else {
       // Non-sports market logic.
+      // Ignore unresolvable prices where the orderbook doesn't provide relevant information.
+      if (!proposal.proposedPrice.eq(ethers.BigNumber.from(0)) && !proposal.proposedPrice.eq(ONE_SCALED)) {
```

Comment:
what about 50/50?

Reply:
Handling 50/50 would require additional logic to flag buys or sells of both sides of the CLOB tokens for less than or more than `$0.5`, using a threshold similar to the one we had before.
Perhaps this could be addressed in a subsequent PR if we decide to support it. However, since the original logic didn’t support it either, I removed the case.